action.yml

```diff
@@ -30,6 +30,10 @@ inputs:
     description: 'Label to be applied to issues created for stale branches.'
     required: false
     default: 'stale branch 🗑️'
+  compare-branches:
```

Comment:
@pohnemus what functionality do we want to add around this? i believe we only care about a branch that is ahead of or diverged from the default branch. branches behind/identical in comparison to the repos default branch wouldn't be valuable. 

we may want to switch it from boolean to allow for something like `info`, `warn`, `error`, or use it to enhance `tag-committer`.

![image](https://user-images.githubusercontent.com/26232872/157163148-7aeb0025-f8d7-409d-a32c-cd05f2172f2a.png)

Reply:
Yep, I agree with your thoughts on what we'd want to be notified about.